projects/media-viewer/src/lib/annotations/annotation-set/annotation-set.component.ts

```diff
@@ -127,14 +127,24 @@ export class AnnotationSetComponent implements OnInit, OnDestroy {
   async createTextHighlight(highlight: Highlight) {
     if (highlight.page === this.page) {
       if (window.getSelection()) {
+        const localElement = (<HTMLElement>highlight.event.target) || (<HTMLElement>highlight.event.srcElement);
+
+        if (localElement.parentElement.childNodes) {
+          localElement.parentElement.childNodes.forEach(child => {
+            child['style']['padding'] = 0;
+            // regex will be targeting the translate style in string
+            // e.g. scaleX(0.969918) translateX(-110.684px) translateY(-105.274px) will become scaleX(0.969918)
+            const translateCSSRegex = /translate[XYZ]\(-?\d*(\.\d+)?(px)?\)/g;
+            child['style']['transform'] = child['style']['transform'].replace(translateCSSRegex, '');
+          });
+        }
```

Comment:
I would just do:

```
for (const child of localElement.parentElement.childNodes || []) {
  // ...
}
```

Reply:
Type 'NodeListOf<ChildNode>' must have a '[Symbol.iterator]()' method that returns an iterator.

Reply:
:man_facepalming: their array like object doesn't implement Symbol.iterator